site/content/en/docs/handbook/offline.md

```diff
@@ -27,18 +27,18 @@ For offline use on other hosts, one can copy the contents of `~/.minikube/cache`
 ```text
 cache/iso/minikube-v1.0.0.iso
 cache/images/gcr.io/k8s-minikube/storage-provisioner_v1.8.1
-cache/images/k8s.gcr.io/k8s-dns-sidecar-amd64_1.14.13
-cache/images/k8s.gcr.io/k8s-dns-dnsmasq-nanny-amd64_1.14.13
-cache/images/k8s.gcr.io/kubernetes-dashboard-amd64_v1.10.1
-cache/images/k8s.gcr.io/kube-scheduler_v1.14.0
-cache/images/k8s.gcr.io/coredns_1.3.1
-cache/images/k8s.gcr.io/kube-controller-manager_v1.14.0
-cache/images/k8s.gcr.io/kube-apiserver_v1.14.0
-cache/images/k8s.gcr.io/pause_3.1
-cache/images/k8s.gcr.io/etcd_3.3.10
-cache/images/k8s.gcr.io/kube-addon-manager_v9.0
-cache/images/k8s.gcr.io/k8s-dns-kube-dns-amd64_1.14.13
-cache/images/k8s.gcr.io/kube-proxy_v1.14.0
+cache/images/registry.k8s.io/k8s-dns-sidecar-amd64_1.14.13
```

Comment:
the kubernetes version needs to be updated, in this example (not changing anything for 1.14)